docs/2022-07-18-JSON-RPC-Server-API.md

```diff
@@ -192,10 +199,68 @@ If `"reason": "cut-point-rule"`, the `next-states` field contains the next state
         "predicate": {"format":"KORE", "version":1, "term":{}},
         "substitution": {"format":"KORE", "version":1, "term":{}},
       }
-    ]
+    ],
+    "logs": []
+  }
+}
+```
+
+If any logging is enabled, the optional `logs` field will be returned containing an array of objects of the following structure:
+
+```json
+{
+  "tag": "simplification",
```

Comment:
Did you end up adding a separate `function-definition` category? I can't find it in the code, any reason why this is harder than expected?

Reply:
no, as @ehildenb  pointed out, you can recover this data in the frontend, so I thought it's best to keep this simple for now.